# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1381212518 14400
#      Tue Oct 08 02:08:38 2013 -0400
# Node ID e89f6dcb37f09fe3bf0382ef5649a4dab393cad1
# Parent  f5950975a1723b7a6461e2929e7f218b8555afb0
* NEWS: Surround contents with <pre></pre>

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,8 +1,9 @@
+<pre>
 Summary of important user-visible changes for version 4.0:
 ---------------------------------------------------------
 
  ** A new GUI is now available with Octave.
 
  ** Warning states may now be set temporarily, until the end of the
     current function, using the syntax
 
@@ -1465,8 +1466,9 @@ Summary of important user-visible change
       loadimage           spfind     spsumsq
       mark_as_command     spinv      str2mat
       mark_as_rawcommand  spkron     unmark_command
       spatan2             splchol    unmark_rawcommand
       spchol              split
       spchol2inv          splu
 
 See NEWS.3 for old news.
+</pre>
